packages/gcp/src/application/GCPAccount.ts

```diff
@@ -30,6 +34,7 @@ import {
   LookupTableOutput,
   GroupBy,
 } from '@cloud-carbon-footprint/common'
+import { GoogleAuth } from 'google-auth-library'
```

Comment:
nitpick: import line could be moved to line 10

Reply:
You mean above the @google-cloud imports? Or to combine it with that import? I believe GoogleAuth can only be imported from the `google-auth-library` package.

Reply:
i meant above the CCF package import but its not a big deal

Reply:
All good! I can move it